DFC.ServiceTaxonomy.Editor/Recipes/New Job Profiles Archive/241.JobProfileHeaderWorkinghoursdetail.recipe.json

```diff
@@ -0,0 +1,162 @@
+{
+  "name": "JobProfileHeaderWorkinghoursdetail",
```

Comment:
Could you please make all names PascalCase? like JobProfileHeaderWorkingHoursDetail for this one?